src/ephemeris/set_library_permissions.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+'''Tool to set permissions for all datasets of a given Galaxy Data Library'''
+
+import argparse
+import logging as log
+from bioblend import galaxy
+import sys, time, os
+
+from .common_parser import get_common_args
+# Print iterations progress
+def printProgressBar (iteration, total, prefix = '', suffix = '', decimals = 1, length = 100, fill = '█', printEnd = "\r"):
```

Comment:
Sound cool on one hand, but would increase dependencies on the other, maybe when someone else starts using it here?

Reply:
I used [rich](https://github.com/Textualize/rich) in [galaxy-upload](https://github.com/galaxyproject/galaxy-upload) but didn't have to worry about adding a dependency in that case since it was already a dependency of one of the Galaxy libs.

Reply:
![image](https://user-images.githubusercontent.com/86979912/202749623-b9f1f2f2-b111-4bf7-96cd-b0d2030de395.png)

Reply:
Just added that with the last commit :)